aries-site/src/pages/components/layer/index.mdx

```diff
@@ -0,0 +1,157 @@
+import { Button, Notification } from 'grommet';
+import { Example, StyleTable } from '../../../layouts';
+import {
+  LayerAnatomy,
+  LayerExample,
+  LayerCenterExample,
+  LayerNotificationExample,
+  LayerSideDrawerExample,
+  LayerStickyExample,
+  LayerStickyScrollExample,
+  ActionableLayerClose,
+  InformationalLayerClose,
+  LayerTypes,
+} from '../../../examples';
+
+<Example
+  componentName="Layer"
+  docs="https://v2.grommet.io/layer?theme=hpe#props"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/LayerExample.js"
+  figma="https://www.figma.com/file/Ts53TAipMolmsv9DxWG3p0/HPE-Layer-Component?node-id=2%3A22"
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Layer/Layer.js"
+>
+  <LayerExample />
+</Example>
+
+## When to use
+
+A layer should be used to present information or task flows to the user without navigating them away from their current context.
+
+## Anatomy
+
+Basic anatomy and spacing that applies to all layer types.
+
+<Example plain>
+  <LayerAnatomy />
+</Example>
+
+| Label  | Region       | Required? | Purpose                                                | Notes                                                                     |
+| ------ | ------------ | --------- | ------------------------------------------------------ | ------------------------------------------------------------------------- |
+| **1**  | **Header**   | required  | Identifies the layer.                                  | --                                                                        |
+| **1a** | **Title**    | required  | A short, descriptive title identifying the layer.      | Implemented as a Heading level 2, size="small".                           |
+| **1b** | **Subtitle** | optional  | Any additional context or information about the layer. | --                                                                        |
+| **2**  | **Body**     | optional  | Any additional content.                                | --                                                                        |
+| **3**  | **Footer**   | optional  | Contains layer actions.                                | The ordering and aligment of footer actions depends on the type of layer. |
+
+<StyleTable
+  data={[
+    {
+      label: 4,
+      property: 'pad',
+      value: 'medium',
+    },
+    {
+      label: 5,
+      property: 'gap',
+      value: 'medium',
+    },
+    {
+      label: 6,
+      property: 'round',
+      value: 'small',
+    },
+    {
+      label: 7,
+      property: 'elevation',
+      value: 'large',
+    },
+  ]}
+/>
+
+## Types of layers
+
+Different use cases and content warrant different layer types. Learn more about each type below.
+
+<LayerTypes />
+
+## Layer closing behavior
+
+### Mouse
```

Comment:
Segmenting by mouse vs. keyboard does not seem right here. I think we should be segmenting by use case (informational vs. actionable) and then describe the closing behavior via buttons, on click outside or Esc (on click outside and Esc should be thought of as equivalents just as click on button and Enter are equivalents).

Reply:
That makes sense. I'll update accordingly.

Reply:
Done.